ede-php-autoload/class-loader/core.el

```diff
@@ -76,6 +76,15 @@ Example: (ede-php-autoload--get-path-relative-to-ns \"My\\Ns\\My\\Class\" \"My\\
     "/")
    (or extension ".php")))
 
+(defun ede-php-autoload--make-completion (name type full-name)
+  "Create a completion item.
+
+NAME is the name of the completion. TYPE is the type and
```

Comment:
What about writing in this docstring the different values of `type` ? That way we have a reference of all these values